library/couchdb

```diff
@@ -1,14 +1,23 @@
-# maintainer: Joan Touzet <wohali@apache.org> (@wohali)
+# see https://couchdb.apache.org/
+# also the announce@couchdb.apache.org mailing list
 
-latest: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2.1.1: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2.1: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
+Maintainers: Joan Touzet <wohali@apache.org> (@wohali), CouchDB Developers <dev@couchdb.apache.org>
```

Comment:
Looks like it fails to parse the maintainers since "CouchDB Developers" is missing a github handle.

> RFC 2822 error: invalid Maintainers: "CouchDB Developers <dev@couchdb.apache.org>" (expected format "Full Name <contact-email-or-url> (@github-handle) OR Full Name (@github-handle)")

Reply:
OK, we'll remove that for now.

That's unfortunate, since development at Apache is definitely a group effort.

Reply:
For cases like this, it might make sense for us to update to allow just an email there. :thinking:  The github handle is mostly so that @tianon and I know who to ping if an new person submits a PR to make sure the update is authorized.  The handle can also be a github org if that makes it easier (like @apache).